tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1635,11 +1635,11 @@ def random_brightness(image, max_delta, seed=None):
     ...     [[7.0, 8.0, 9.0],      
     ...       [10.0, 11.0, 12.0]]]
     >>> tf.image.random_brightness(x, 0.2)
-    array([[[ 0.91842633,  1.9184263 ,  2.9184263 ],
-        [ 3.9184263 ,  4.9184265 ,  5.9184265 ]],
+    array([[[ 0.91..,  1.91.. ,  2.91.. ],
```

Comment:
This one is fixed -> https://github.com/tensorflow/tensorflow/pull/35388/commits/362df5b9d57c977d665b2a9324b4df8716fd2844
For the dtype thing, should I change all of them with that or just that particular one? And looks like there is no type like tf.int so I guess I will be using tf.int32 which requires me to change values to like "1" instead of "1.0". Is that what you are asking for?

Reply:
Only that one should be enough. As that one changes the dtype, we have to start with a tensor with one dtype and the test will prove that the dtype changed.

Reply:
Something like `x = tf.constant(..., dtype=tf.int32)`

Reply:
Oh I just remembered why I did the examples with float32. The reason was because if I did use `tf.int32`, the examples would be like that and I don't think this is right: 
```
>>> x
<tf.Tensor: shape=(2, 2, 3), dtype=int32, numpy=
array([[[ 1,  2,  3],
        [ 4,  5,  6]],

       [[ 7,  8,  9],
        [10, 11, 12]]])>
>>> tf.image.random_brightness(x, 0.2) 
<tf.Tensor: shape=(2, 2, 3), dtype=int32, numpy=
array([[[-117015968, -117015968, -117015968],
        [-117015968, -117015960, -117015960]],

       [[-117015960, -117015960, -117015960],
        [-117015960, -117015960, -117015952]]])>
```

Reply:
Should I create an issue about this or is this some expected value since I currently can't think of what is going on between the difference for `float32` vs `int32` brightness settings.

Reply:
Oh sorry, I just noticed that you wanted me to do the change on `convert_image_dtype`, I was wondering why would the test will prove the dtype change :D But now, if I do like this, all output is 0, is this expected again?
```
>>> x
<tf.Tensor: shape=(2, 2, 3), dtype=int32, numpy=
array([[[ 1,  2,  3],
        [ 4,  5,  6]],

       [[ 7,  8,  9],
        [10, 11, 12]]])>
>>> tf.image.convert_image_dtype(x, dtype=tf.float16, saturate=False) 
<tf.Tensor: shape=(2, 2, 3), dtype=float16, numpy=
array([[[0., 0., 0.],
        [0., 0., 0.]],

       [[0., 0., 0.],
        [0., 0., 0.]]], dtype=float16)>
```

Reply:
That doesn't sound right. Can you file a bug please and assign to me?

In the meanwhile, let's use float32 to float16.

Reply:
Done, but couldn't assign you as explained in the issue as well. Issue -> https://github.com/tensorflow/tensorflow/issues/35450

Reply:
Thank you